### ISSUE TITLE
User Default for Pixel Height

### ISSUE THREAD

**Issue by HoraceAndTheSpider** [HoraceAndTheSpider/UAEConfigMaker#47]
allow users to select a default value for the above in `hostconfig`

**Comment by ghost** [comment]
Doing this though will need more entries in the Screen Height files because if for instance they change it to 256 default, there might not be an entry for said game because the original default 240 was correct.

**Comment by HoraceAndTheSpider** [comment]
Yes, users will have to take that into account if they choose to use alternative values.
